datafusion/sqllogictest/bin/sqllogictests.rs

```diff
@@ -39,6 +40,23 @@ pub fn main() -> Result<()> {
         .block_on(run_tests())
 }
 
+fn value_validator(actual: &[Vec<String>], expected: &[String]) -> bool {
+    let expected = expected
+        .iter()
+        // Trailing whitespace from lines in SLT will typically be removed, but do not fail if it is not
+        // If particular test wants to cover trailing whitespace on a value,
```

Comment:
This is probably a good thing to add in the user documentation (aka https://github.com/apache/datafusion/blob/main/datafusion/sqllogictest/README.md)

I think it would be hard to find here for someone writing sqllogictests

Reply:
it feels so edge-case'y. but yes, we can add

Reply:
I will make a follow on PR

Reply:
https://github.com/apache/datafusion/pull/13215